packages/pg/lib/client.js

```diff
@@ -122,6 +124,8 @@ class Client extends EventEmitter {
   static connectionMap = new Map()
   // Map of failedHost -> ServerInfo of host
   static failedHosts = new Map()
+  // Map of failedHost -> Time at which host was added to failedHosts Map
+  static failedHostsTime = new Map()
```

Comment:
We should try to put this (and other server-specific info) inside the `ServerInfo` itself going forward.